docker/perf_layer_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
nit: `#!/usr/bin/env bash`. The preferred bash may not be in `/bin` on all systems. For example, macOS ships a very old version of bash and the user-installed one is typically in `/usr/local/bin/`